v1.16/Dockerfile

```diff
@@ -13,42 +18,69 @@ RUN apk update \
         ruby ruby-irb ruby-etc \
         tini libmaxminddb geoip \
         snappy \
- && buildDeps=" \
-      make gcc g++ libc-dev \
-      wget bzip2 zlib-dev git linux-headers \
-      automake autoconf libtool build-base \
-      ruby-dev libc6-compat geoip-dev \
-      snappy-dev \
-    " \
  && apk add --no-cache --virtual .build-deps \
         build-base \
         ruby-dev gnupg \
- && apk add $buildDeps \
+ && apk add $BUILD_DEPS \
  && echo 'gem: --no-document' >> /etc/gemrc \
- && gem install bundler && bundle install \
- && apk del .build-deps $buildDeps \
+ # The only required fluentd dependency is the label router
+ # kubeclient install is upgrading the one from the base image
+ && gem install specific_install -v 0.3.8 \
+ && gem specific_install -l https://github.com/kube-logging/fluent-plugin-label-router.git --ref 2ff43789f895735b7852e0a1a809280e22d9e8ef \
+ && gem specific_install -l https://github.com/ManageIQ/kubeclient.git --ref 054bff2c5e31a555004be2b3c4d32fb9dc5e6a0f \
```

Comment:
seems to be already part of https://github.com/SumoLogic/sumologic-kubernetes-fluentd/blob/main/alpine.Dockerfile#L32 with a different hash. do we need this one?